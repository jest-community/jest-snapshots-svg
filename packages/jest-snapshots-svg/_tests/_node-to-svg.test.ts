import * as yoga from "yoga-layout"

import nodeToSVG from "../node-to-svg"

const component = (name) => ({
  type: name,
  props: {},
  children: [],
  textContent: undefined,
  layout: {
    left: 0,
    right: 6,
    top: 0,
    bottom: 100,
    width: 600,
    height: 400
  }
})

describe("nodeToSVG", () => {
    it("handles a simple square", () => {
      const rootNode = component("my component")

      const settings = {
        width: 1024,
        height: 768,
      }

      const results = nodeToSVG(0, rootNode, settings)
      expect(results).toMatchSnapshot()
    })
})
