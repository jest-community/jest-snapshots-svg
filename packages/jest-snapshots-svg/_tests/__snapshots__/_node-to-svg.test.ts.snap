// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`nodeToSVG handles a simple square 1`] = `
"
 <rect type=\\"my component\\" fill-opacity=\\"0.1\\" stroke-width=\\"1\\" stroke=\\"black\\" x=\\"0\\" y=\\"0\\" width=\\"600\\" height=\\"400\\"/>"
`;
